README.md

```diff
@@ -147,6 +147,7 @@ Communications (Website, etc.)|Every other week on Monday at 10:00 PT|[Google Do
 End-User WG|Every other week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1FqEo2PfHPP8iZX_W-C3YgXFUjhfwmAhoREUE8ZJPJfc/)||
 eBPF|Every week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 Agent Management WG|Every other week on Tuesday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4)|[Slack](https://cloud-native.slack.com/archives/C02J58HR58R)|
+Prometheus Interoperability WG|Every week on Wednesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8/edit#)|[Slack](https://cloud-native.slack.com/archives/C01LSCJBXDZ)|
```

Comment:
@open-telemetry/wg-prometheus it looks like meetings are now every other week? just confirming, and I will update.

also, the doc hasn't been transferred to cncf-opentelemetry-governance@lists.cncf.io yet, I've added that as a TODO in #1147

Reply:
Yes, meetings are currently every other week

Reply:
```suggestion
Prometheus Interoperability WG|Every other week on Wednesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8/edit#)|[Slack](https://cloud-native.slack.com/archives/C01LSCJBXDZ)|
```